pkg/cmd/pulumi/deployment_settings_config.go

```diff
@@ -251,6 +260,18 @@ func initStackDeploymentCmd(
 		return err
 	}
 
+	oidcExplanationMsg := `
+Pulumi supports OpenID Connect (OIDC) integration across various services by
+leveraging signed, short-lived tokens and eliminating the necessity for hardcoded
+cloud provider credentials and facilitates the exchange of these tokens for
+short-term credentials.
```

Comment:
Should we leave this message on a single line and let the terminal to the wrapping? Similarly for the other messages below.

Reply:
sure, I thought for such long messages it was easier to read incases the terminal is to wide (and in some cases some messages are formated this way here). Will change

Reply:
done